libautoscoper/src/PSO.cpp

```diff
@@ -123,6 +161,40 @@ float getRandomClamped()
   return (float)rand() / (float)RAND_MAX;
 }
 
+#ifdef Autoscoper_COLLISION_DETECTION
+void checkCollision(Particle& p,
+                    std::vector<float>& avgNonCollidedPosition,
+                    std::vector<float>& avgCollidedPosition,
+                    unsigned int& collidedCount)
```

Comment:
`unsigned int&` - > `unsigned int`

Reply:
Looks like `collidedCount` is being modified in this function, so the by-by-reference is intentional?

Reply:
Good catch. I missed that, I am marking this comment as resolved.